### DISCUSSION TITLE
Increasing Toolbar Icon Size in DBeaver

### DISCUSSION THREAD

**Discussion by Suniron** [dbeaver/dbeaver#21596] [Q&A] [answered]
Hello,

I'm using DBeaver for my database management tasks and I find the toolbar icons to be a bit small for my liking. I've looked through the settings but couldn't find an option to increase the size of these icons. Could you please guide me on how to achieve this? Any help would be greatly appreciated.

Thank you.

Screenshot on my 4K screen (it's hard to read with a 27" monitor):
![image](https://github.com/dbeaver/dbeaver/assets/36677774/5d349c12-febb-49e3-9f51-d4e65987a1b5)

**Comment by E1izabeth** [comment, marked as answer]
You could try this https://github.com/dbeaver/dbeaver/issues/20704#issuecomment-1700730969

**Reply by Suniron** [reply]
It works, thanks!

**Reply by Vatsal-Soni-codeware** [reply]
Yes it works in Win11. Thanks.

**Reply by DanielTuran** [reply]
It does not work when installed from Windows Store. The installation folder is protected and access is denied even for administrators

**Reply by boudhayangit** [reply]
Worked for me. Notepad++ did ask me to reopen in Admin mode when I tried to save the updated ini file, after which everything went per plan.

**Reply by DanielTuran** [reply]
If you are logged in as regular user, not admin, then event if you start it as admin, it does not work. Anyway, those files (Program Files) are not supposed to be edited at all.
